packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -139,11 +125,7 @@ const _ActionListSection = ({
       >
         {children}
       </BaseBox>
-      {_hideDivider && isReactNative() ? null : (
-        <Box marginX="spacing.3" marginY="spacing.1">
-          <ActionListSectionDivider />
-        </Box>
-      )}
+      {_hideDivider && isReactNative() ? null : <Divider marginX="spacing.3" marginY="spacing.1" />}
```

Comment:
🤔 how did this fix the bug? I can see no logic changes.

Reply:
this didn't fix the bug. This is just refactor to remove additional wrapper and nesting of divider. Adding `role="separator"` fixed the bug. More info in PR description